Sources/Date+Sweet.swift

```diff
@@ -0,0 +1,5 @@
+public extension Date {
+    public func components(_ components: Set<Calendar.Component>) -> DateComponents {
+        return Calendar.current.dateComponents(components, from: self)
```

Comment:
worth making a version of this where the `calendar` is passed in as a default parameter so if for some reason you need to use another calendar, you can?